charts/tempo/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: tempo
 description: Grafana Tempo Single Binary Mode
 type: application
-version: 0.7.4
-appVersion: v1.0.1
+version: 0.7.5
+appVersion: v1.0.2
```

Comment:
let's keep this at 1.0.1. this should match the actual tempo version which will remain 1.0.1.